gui/elements/qt/_controls.py

```diff
@@ -1,10 +1,13 @@
 from PyQt5.QtCore import Qt
-from PyQt5.QtWidgets import QHBoxLayout, QSlider
+from PyQt5.QtWidgets import QHBoxLayout, QWidget
 
+from napari_gui.util.range_slider import QVRangeSlider
```

Comment:
why is import include `napari_gui`, in other places havn't we just used `.`

Reply:
Good catch! All imports within the package must be relative for it to work with the Napari plugin core.